compiler/rustc_span/src/lib.rs

```diff
@@ -1725,6 +1725,8 @@ pub struct SourceFile {
     pub start_pos: BytePos,
     /// The byte length of this source.
     pub source_len: RelativeBytePos,
+    /// The byte length of this source before normalization.
+    pub original_source_len: u32,
```

Comment:
Suggestion: I'm not sure how invasive the change is, but we can rename this pair of source lengths (better names welcome):

- `source_len` -> `normalized_source_len`
- `original_source_len` -> `unnormalized_source_len`

When reading this diff, I find that `source_len` vs `original_source_len` is really not obvious. I can appreciate if the diffs is intentionally made small to make review easier, but I would prefer if we also do a rename for the `source_len` field for consistency -- this is really not obvious.

Reply:
Nice suggestion. Done in a separate commit.